test/python-bdk.spec.js

```diff
@@ -38,6 +38,13 @@ describe('Python BDK', () => {
         let privateKey = fs.readFileSync(path.join(dir, 'rsa/privatekey.pem'), 'utf-8')
         let generatedPublicKey = fs.readFileSync(path.join(dir, 'rsa/publickey.pem'), 'utf-8')
         assertKeyPair(privateKey, generatedPublicKey)
+
+        let requirement = fs.readFileSync(path.join(dir, 'requirements.txt'), 'utf-8')
+        let config = fs.readFileSync(path.join(dir, BASE_RESOURCE, 'config.yaml'), 'utf-8')
+        assert(requirement.includes('sym-api-client-python>='))
```

Comment:
we could also test that there is a version set (using a regex maybe to not be dependant on the version), just to make sure that the templating happened